concrete/src/Permission/Response/AreaResponse.php

```diff
@@ -34,6 +34,31 @@ public function canAddLayout()
     {
         return $this->validate('add_layout_to_area');
     }
+    public function canApproveAreaVersions()
+    {
+        return $this->validate('approve_area_versions');
+    }
+    public function canDeleteArea()
```

Comment:
@mlocati @aembler Would it be possible to add a release number to new methods? That would make it easier for package developers to use these methods, without risking backward compatibility issues. Similarly to e.g. https://github.com/WordPress/wordpress-develop/blob/811eef33a31aab13d38a13522d4a58419566225b/src/wp-includes/user.php#L25

Reply:
I think it's a great idea. Sure, we have https://documentation.concrete5.org/api , but it's annoying to look at those pages every time we use a method.

Reply:
That is a handy feature - just wondering how would that work for things that haven't been released yet? A -dev tag that would need to be updated?

Also, 8.4.2 isn't included in the API docs :(

Reply:
> Also, 8.4.2 isn't included in the API docs :(

I guess that the core team didn't implement what I suggested at https://github.com/concrete5/api/pull/5 and they prefer updating the API docs manually...